lib/Requestable.js

```diff
@@ -152,7 +155,13 @@ class Requestable {
     */
    _request(method, path, data, cb, raw) {
       const url = this.__getURL(path);
-      const headers = this.__getRequestHeaders(raw);
+
+      const AcceptHeader = (data || {}).AcceptHeader;
```

Comment:
I think @clayreimann is getting at the usage of PascalCasing instead of camelCasing.